### PR TITLE
Feature/pdfmerger

### PR DESCRIPTION
**Pull request by jemten**
### This PR adds | fixes:

- merge arriba reports with pdfmerger 

### How to test:

- Automatic and continuous test suite

### Expected outcome:
- [ ] Installation, unit and integration test suite pass

### Review:
- [ ] Code review
- [ ] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [ ] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: lib/MIP/Recipes/Analysis/Fusion_report.pm [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+
+    # Set the version for version checking
+    # Functions and variables which can be optionally exported
+    our @EXPORT_OK = qw{ analysis_fusion_report };
+
+}
+
+sub analysis_fusion_report {
+
+## Function : DESCRIPTION OF RECIPE
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Analysis/Fusion_report.pm:35]: Add a description

File: lib/MIP/Recipes/Analysis/Fusion_report.pm [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+## Returns  :
+## Arguments: $active_parameter_href   => Active parameters for this analysis hash {REF}
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Analysis/Fusion_report.pm:37]: Align fat comma

File: lib/MIP/Recipes/Analysis/Fusion_report.pm [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+                attribute        => q{sample_display_name},
+                sample_id        => $sample_id,
+                sample_info_href => $sample_info_href,
+            }
+        );
+        if ( $sample_display_name and not $use_sample_id_as_display_name ) {
+
+            $report_path =
+              catfile( $outdir_path, $sample_display_name . $UNDERSCORE . q{fusions.pdf} );
+        }
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Analysis/Fusion_report.pm:220]: Sometime in the future we should make this a generic sub that one can import and reuse

**Review by henrikstranneheim — APPROVED** [review]
👍

File: lib/MIP/Recipes/Analysis/Fusion_report.pm (hunk 3 above)
Comment (jemten) [reply lib/MIP/Recipes/Analysis/Fusion_report.pm:220]: Agreed

**Pull request merged**